backend/api/v1/endpoints/research.py

```
@@ -65,11 +66,11 @@
         return {
             "message": "Paper uploaded successfully",
             "paper_id": new_paper.id,
-            "file_path": str(file_path),
+            "file_name": safe_filename,
         }
 
     except Exception as e:
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)